db/seeds.rb

```diff
@@ -42,4 +42,9 @@
     end
   end
 
+# Create dummy teachers
+%w[Sarah Bob Pomona].each do |first_name|
+  Teacher.find_or_create_by!(first_name:)
+end
```

Comment:
I wonder if we could/should start using factories in this seeds file? 🤔

Reply:
I still want to specify the name of a teacher/mentor here don't want to use faker to generate random names every time. So I don't really see a need to use factories here

Reply:
Cool cool. I don't have a strong opinion either way.